sdk/js/src/index.ts

```diff
@@ -147,6 +154,10 @@ export class Client {
             ...message.status.bid_status,
           });
         }
+      } else if ("type" in message && message.type === "svm_chain_update") {
+        if (this.websocketSvmChainUpdateCallback !== undefined) {
```

Comment:
nit: a better check would be `typeof this.websocketSvmChainUpdateCallback === 'function'`

Reply:
done